src/main.c

```diff
@@ -726,6 +770,13 @@ static int listener_setup_ssl(h2o_configurator_command_t *cmd, h2o_configurator_
         ERR_print_errors_cb(on_openssl_print_errors, stderr);
         goto Error;
     }
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+    if (cipher_suite_tls13 != NULL && SSL_CTX_set_ciphersuites(ssl_ctx, (*cipher_suite_tls13)->data.scalar) != 1) {
+        h2o_configurator_errprintf(cmd, *cipher_suite, "failed to setup TLS 1.3 cipher suite\n");
+        ERR_print_errors_cb(on_openssl_print_errors, stderr);
+        goto Error;
+    }
+#endif
```

Comment:
I think we do not need this, because we consistently use picotls for handling TLS 1.3 connections.